core/src/main/java/tc/oc/pgm/api/integration/FriendIntegration.java

```diff
@@ -0,0 +1,16 @@
+package tc.oc.pgm.api.integration;
+
+import org.bukkit.entity.Player;
+
+/** FriendIntegration - Determine whether two players are friends * */
+public interface FriendIntegration {
```

Comment:
These are definetly *not* functional interfaces. They're interfaces which just happen to have a single method... for now. Nothing prevents them getting more methods in the future and we shouldnt set an interface that says otherwise